crates/webidl/src/lib.rs

```diff
@@ -244,6 +244,8 @@ fn immutable_slice_whitelist() -> BTreeSet<&'static str> {
         "clearBufferfv",
         "clearBufferiv",
         "clearBufferuiv",
+        // WebSocket
+        "send_with_u8_array",
```

Comment:
I am not sure but I *think* it requires *camelCase*?

Reply:
Hm yeah it looks like the others are all camel-case here. @chinedufn mind testing this out locally to see if it had the desired effect on the method?

Reply:
Thanks! https://github.com/chinedufn/wasm-bindgen/commit/001f4b08844b12f8a110a1bcd248316f49b4c8f9#diff-266f7f5e4a4a2bbbf9377f3d19673606R95-R98